src/components/FormField/index.tsx

```diff
@@ -27,30 +27,48 @@ const components: {
  * > This component is not found in Material UI.
  */
 export const FormField = React.forwardRef(({
-    type = 'text',
+    helperText,
     label,
     name,
+    onChange,
+    type = 'text',
     validation = {},
-    helperText,
     ...otherProps
 }: Props, ref: React.Ref<any>): React.ReactElement => {
 
 
     const classes = useStyles();
+    const { trigger } = useForm();
```

Comment:
I would think that you would get this off of context since it's wrapped in useForm already